sdk/monitor/monitor-opentelemetry/CHANGELOG.md

```diff
@@ -1,14 +1,12 @@
 # Release History
 
-## 1.0.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 1.1.0 (2023-10-09)
```

Comment:
Missing several changes here
https://github.com/Azure/azure-sdk-for-js/pull/27337
https://github.com/Azure/azure-sdk-for-js/pull/27330
https://github.com/Azure/azure-sdk-for-js/pull/27317
https://github.com/Azure/azure-sdk-for-js/pull/27179
https://github.com/Azure/azure-sdk-for-js/pull/27163
https://github.com/Azure/azure-sdk-for-js/pull/27161
https://github.com/Azure/azure-sdk-for-js/pull/27145

Reply:
Ah, these didn't pop up in my issue search, my bad.

Reply:
Added #27337 and #27330. #27163, #27161, and #27145 appear to be a part of the last release.